### PR TITLE
Initial port of bylaws from Google Docs into Markdown - started with …

### PR DESCRIPTION
**Pull request by adamfast**
…a conversion plugin and then did a bunch of manual edits.

### CODE REVIEW

File: bylaws.md [hunk 1]
```diff
@@ -0,0 +1,602 @@
+<center>
+#Bylaws
+
+#of
+
+#DJANGO EVENTS FOUNDATION NORTH AMERICA
```
Comment (drewbrew) [inline bylaws.md:6]: Was this supposed to be three separate lines?

File: bylaws.md [hunk 2]
```diff
@@ -0,0 +1,602 @@
...
+
+#A California Public Benefit Corporation
+</center>
```
Comment (drewbrew) [inline bylaws.md:9]: Stray HTML?

File: bylaws.md [hunk 3]
```diff
@@ -0,0 +1,602 @@
...
+
+(a) Perform any and all duties imposed on them collectively or individually by law, by the articles of incorporation of this corporation, or by these bylaws;
+
+(b) Appoint and remove, employ and discharge, and, except as otherwise provided in these bylaws, prescribe the duties and fix the compensation, if any, of all officers, agents, and employees of the corporation;
+
+(c) Supervise all officers, agents, and employees of the corporation to assure that their duties are performed properly;
+
+(d) Meet at such times and places as required by these bylaws;
+
+(e) Register their addresses with the secretary of the corporation and notices of meetings mailed or telegraphed to them at such addresses shall be valid notices thereof.
```
Comment (drewbrew) [inline bylaws.md:59]: Should we convert these to markdown lists?

**Review by drewbrew — COMMENTED** [review]
Thanks for doing this!

File: bylaws.md (hunk 2 above)
Comment (jefftriplett) [reply bylaws.md:9]: It's centered in our original.

**Comment by KatherineMichel** [comment]
Thank you all for working on this!

File: bylaws.md (hunk 3 above)
Comment (jefftriplett) [reply bylaws.md:59]: Let's follow up on this if you want to create a new issue. Markdown tends to auto-renumber which we probably don't want for finding stuff quickly, but maybe I'm missing how you want to structure it.

Comment (drewbrew) [reply bylaws.md:59]: No, this is totally fine. It's not worth changing if you're good with the manual list.

**Pull request merged**